.github/workflows/lint.yml

```diff
@@ -9,14 +9,40 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
+    - uses: actions/setup-go@v3
       with:
-        go-version: 1.18.x
+        go-version: 1.19.x
+
+    - name: Get Go cache paths
```

Comment:
@Aoang curious why you ended up doing this? In wazero, we're noticing the cache doesn't seem to work for golangci-lint and wondering if there's some difference between this and hard-coding (considering it is only run with ubuntu). Any thoughts appreciated!

https://github.com/tetratelabs/wazero/blob/main/.github/workflows/commit.yaml#L46-L52

Reply:
continue https://github.com/tetratelabs/wazero/pull/804